sycl/include/sycl/ext/oneapi/matrix/matrix-intel.hpp

```diff
@@ -0,0 +1,347 @@
+//==------------------ matrix-intel.hpp - SYCL matrix ----------*- C++ -*---==//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+// ===--------------------------------------------------------------------=== //
+
+#pragma once
+
+#include "matrix-unified-utils.hpp"
+#include <CL/__spirv/spirv_ops.hpp>
+#include <sycl/detail/defines_elementary.hpp>
+#include <sycl/feature_test.hpp>
+
+namespace sycl {
+__SYCL_INLINE_VER_NAMESPACE(_V1) {
+namespace ext {
+namespace intel::experimental::matrix::layout {
+constexpr sycl::ext::oneapi::experimental::matrix::layout packed =
```

Comment:
Why are we introducing the layout in a new namespace here?

Reply:
Correct I am not fully up to date with joint matrix atm! OK thanks for explanation, that makes sense to me.